src/main/resources/application-example.properties

```diff
@@ -0,0 +1,10 @@
+#--------------------- DB Connection ------------------
+spring.datasource.url=
+spring.datasource.username=
+spring.datasource.password=
+
+#--------------------JPA-ORM Properties-----------------
```

Comment:
This should be in your main properties file, is there any reason why you moved this piece here?

Reply:
Done.